Example/AblyChatExample/Mocks/MockSubscriptions.swift

```diff
@@ -0,0 +1,76 @@
+import Ably
+import AblyChat
+
+struct MockMessageSubscription: Sendable, AsyncSequence, AsyncIteratorProtocol {
+    typealias Element = Message
+    
+    let clientID: String
+    let roomID: String
+    
+    public init(clientID: String, roomID: String) {
+        self.clientID = clientID
+        self.roomID = roomID
+    }
+    
+    public mutating func next() async -> Element? {
+        try? await Task.sleep(nanoseconds: 2 * 1_000_000_000)
+        return Message(timeserial: "\(Date().timeIntervalSince1970)",
```

Comment:
Not sure if this is the correct format for `timeserial`, but we can figure it out later.